
# HG changeset patch
# User andrew
# Date 1384279115 0
# Node ID 9d01164b2dfc36fb13984cb0854b2c7a9e4dbfbe
# Parent  62c7dcedbbc4b48057f7aa98b5628b6d691aa656
Include defs.make in vm.make so VM_LITTLE_ENDIAN is defined on Zero builds

diff -r 62c7dcedbbc4 -r 9d01164b2dfc make/bsd/makefiles/vm.make
--- openjdk/hotspot/make/bsd/makefiles/vm.make	Wed Aug 07 19:02:06 2013 +0400
+++ openjdk/hotspot/make/bsd/makefiles/vm.make	Tue Nov 12 17:58:35 2013 +0000
@@ -27,6 +27,7 @@
 
 # Common build rules.
 MAKEFILES_DIR=$(GAMMADIR)/make/$(Platform_os_family)/makefiles
+include $(GAMMADIR)/make/defs.make
 include $(MAKEFILES_DIR)/rules.make
 include $(GAMMADIR)/make/altsrc.make
 
diff -r 62c7dcedbbc4 -r 9d01164b2dfc make/linux/makefiles/vm.make
--- openjdk/hotspot/make/linux/makefiles/vm.make	Wed Aug 07 19:02:06 2013 +0400
+++ openjdk/hotspot/make/linux/makefiles/vm.make	Tue Nov 12 17:58:35 2013 +0000
@@ -27,6 +27,7 @@
 
 # Common build rules.
 MAKEFILES_DIR=$(GAMMADIR)/make/$(Platform_os_family)/makefiles
+include $(GAMMADIR)/make/defs.make
 include $(MAKEFILES_DIR)/rules.make
 include $(GAMMADIR)/make/altsrc.make
 

